docs/diagrams/SummariseFeatureActivityDiagram.puml

```diff
@@ -1,19 +1,22 @@
 @startuml
 'https://plantuml.com/activity-diagram-beta
 
-(*)--> "User inputs Summarise Command without any tags"
--right-> "Tracey collates total number of students with covid"
---> "Tracey collates number of students in each\nhall block under each covid status category"
--left-> "Tracey collates number of\ncovid positive students in each faculty"
+start
+:User input summarise command;
```

Comment:
Perhaps could add in the part whereby the user put in an irrelevant parameter for this command

Reply:
Definitely. Thanks for highlighting this as I have missed it.